p2p/protocol/identify/id.go

```diff
@@ -395,28 +399,35 @@ func (ids *IDService) sendIdentifyResp(s network.Stream) {
 	}
 
 	ph.snapshotMu.RLock()
-	mes := &pb.Identify{}
-	ids.populateMessage(mes, c, ph.snapshot)
-	w := ggio.NewDelimitedWriter(s)
-	w.WriteMsg(mes)
-
+	ids.writeChunkedIdentifyMsg(c, ph.snapshot, s)
 	log.Debugf("%s sent message to %s %s", ID, c.RemotePeer(), c.RemoteMultiaddr())
 }
 
 func (ids *IDService) handleIdentifyResponse(s network.Stream) {
 	c := s.Conn()
 
-	r := ggio.NewDelimitedReader(s, 2048)
+	r := ggio.NewDelimitedReader(s, signedIDSize)
 	mes := pb.Identify{}
 	if err := r.ReadMsg(&mes); err != nil {
 		log.Warning("error reading identify message: ", err)
 		s.Reset()
 		return
 	}
 
+	if mes.More {
```

Comment:
I'm pretty sure we don't need "more". We can just read till the end.

Reply:
This has been removed now since merging the messages works well.

Reply:
We can remove the more field now, correct? I've gone ahead and done that but I'd like to double check with you (we can revert and add it back if you left it there for a reason).

Reply:
@Stebalien Yes, we can remove it. Sorry, I forgot this bit.